FullBackgroundAudio/app/src/main/java/com/jwplayer/demo/fullbackgroundaudio/App.java

```diff
@@ -0,0 +1,34 @@
+package com.jwplayer.demo.fullbackgroundaudio;
+
+import android.app.Application;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
+
+public class App extends Application {
+    //Notification related properties
+    public static final String CHANNEL_ID = "JW";
+    private static final String CHANNEL_NAME = "JW Notifications";
+    private static final String CHANNEL_DESCRIPTION = "This is the default notification channel for the JW Player";
+    private static final int CHANNEL_IMPORTANCE = NotificationManager.IMPORTANCE_LOW;
```

Comment:
Just wondering the priority used here, for background playback controls, should the priority be low, or just use the normal priority

Reply:
The priority used here defines how disruptive the notification is to the user, if the notification should pop up in the screen, make noise, vibrate, etc. I checked for similar media player apps and this is the general behavior.